content/changelog/2024-11-29.md

```diff
@@ -0,0 +1,108 @@
+
+The PostgreSQL Global Development Group has released an [out-of-cycle minor version update](https://www.postgresql.org/about/news/out-of-cycle-release-scheduled-for-november-21-2024-2958/) to address regressions introduced in the previous minor version update. While Neon was unaffected by these regressions, we prioritize staying up to date with the latest version. As a result, we have updated our supported PostgreSQL versions to [14.15](https://www.postgresql.org/docs/release/14.15/), [15.10](https://www.postgresql.org/docs/release/15.10/), [16.6](https://www.postgresql.org/docs/release/16.6/), and [17.2](https://www.postgresql.org/docs/release/17.2/).
+
+When a new minor version is available on Neon, it is applied the next time your compute restarts. For more about how we handle Postgres version upgrades, refer to our [Postgres version support policy](/docs/postgresql/postgres-version-policy).
+
+And don't forget to check out this week's fixes and improvements:
+
+<details>
+
+<summary>**Fixes & improvements**</summary>
+
+- **Logical Replication**
+
+  Neon now automatically removes **inactive** replication slots (if other **active** slots exist) after approximately 40 hours, up from the previous 75 minutes. This change reduces the risk of unexpected slot removal. If you've implemented measures to prevent slots from becoming inactive, you can now relax those measures accordingly.
+
+- **Neon Serverless Driver**
+
+  - Fixed an issue with insertion of Buffer and ArrayBuffer values for `BYTEA` fields over HTTP. Thanks to [@andyjy](https://github.com/andyjy) for the fix.
+  - Fixed an authentication error that occurred when passing the `authToken` property only on the `sql` function.
+
+  For the latest improvements and fixes for the Neon Serverless Driver, refer to the [Neon Serverless Driver Changelog](https://github.com/neondatabase/serverless/blob/main/CHANGELOG.md).
+
+- **.NET support**
+
+  We've added .NET to the list of supported connection strings for various languages and frameworks in the Dashboard. You can now find connection details for .NET in both the connection widget and the Quickstart.
+
+  For more information on connecting to Neon with .NET, see [Connect a .NET (C#) application to Neon](/docs/guides/dotnet-npgsql).
+
+- **Console updates**
+
+  - We've added a **Monitor** button to each listed endpoint on your Branch details page. Click the button to open the **Monitoring** page, displaying metrics for this endpoint.
+
+    ![monitor button on endpoint item](/docs/relnotes/monitor_button_endpoint_item.png)
+
+  - Added quick compute size editing directly from the Branches page — just click the size link in the primary branch column to adjust your settings.
+
+    ![branches table compute drawer](/docs/relnotes/branches_table_compute_drawer.png)
+
+  - We've modified our autoscaling range slider to better handle larger compute sizes and added a constraint: your maximum range can be no more than 8x your minimum.
```

Comment:
remove this item. It's still under a feature flag, only visible to Neon employees.

Reply:
removed